model-mesh/runtimes/triton-2.x.yaml

```diff
@@ -49,7 +49,7 @@ spec:
 
   containers:
     - name: triton
-      image: tritonserver-2:replace
```

Comment:
@VedantMahabaleshwarkar You changed `tritonserver-2:replace` in this file and the [triton-2](https://github.com/opendatahub-io/odh-manifests/blob/master/model-mesh/runtimes/kustomization.yaml#L20) imagetransform no longer works